modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml

```diff
@@ -11,13 +11,14 @@
 	<version>${version}</version>
 	<profiles>
 		<profile>
+			<id>standalone</id>
 
-			<!-- Enable this profile in standalone -->
+			<!-- Uncomment the following element if project is not in Workspace
 
-			<id>standalone</id>
 			<activation>
-				<activeByDefault>false</activeByDefault>
-			</activation>
+				<activeByDefault>true</activeByDefault>
+				<
```

Comment:
Is this a typo?

Reply:
That is correct. We needed the switch the logic around.

Reply:
Nm! i see the weird `<`. Removing